src/pages/en/utol/lecturers/settings/course_participants/index.md

```diff
@@ -0,0 +1,210 @@
+
+4. Enter the course participants’ information in the format.
+    * The format includes fields for the User ID, the Student ID number, the Authority (required), and the Timetable code.
+    * Either the User ID or the Student ID number is required when registering students. The User ID is required when registering faculty and staff members.
+    * Select either “Course Instructor”, “TA”, “Course Designer”, or “Enrolled Student” from the pull-down menu in the Authority field. If you would like to register as “TA and Enrolled Student”, please enter each authority in separate lines.
+    * “Timetable code” is an alphanumeric code displayed between the “course organization name” and the “course name” at the top page of the relevant course page.
+    * If you try to register a user who is already registered for the course with the same authority, an error may occur depending on how the user was originally registered
+        * An error will occur when trying to register a participant using the “Instructor search”
+        * The registration method will be [updated to “Instructor search”](#update) when you try to register using “Self-registration” or “Favorite registration”.
+        * No error will occur when using the “Course registration”. However, after the data is linked with the UTAS the following morning, the registration method will revert to “Course registration”.
+
+5. Select a file from “Reference” in the “Batch registration” field and upload it.
+
+    ![](upload-file.png){:.medium}
+
+    * If the upload is successful, the information of newly registered users will be displayed in red on the “Course participants list’ screen.
+    * If an input error occurs, no data in the upload files will be processed. After fixing the error, you will need to re-upload all the data.
+    * Please note that, at this point, no additional course participants have been registered yet.
+
+6. Click “Proceed to confirmation screen”
+
+    ![](confirm.png){:.small}
+
+7. The users (authority) to be added are displayed. Please review the changes and then click the “Register” button.
+
+    ![](confirm-registration.png){:.medium}
+
+## Updating the method to register course participants
+{:#update}
+
+In addition to cancellation [due to deletion of course participants](#delete), course registration for “Enrolled Students” may be canceled due to the following events,.
+
+* When users who registered using “Self-registration” cancel their registration through UTOL, or when the registration period ends.
+* When users who registered using “Course registration” cancel through UTAS.
+* When users who registered using “Favorite registration” cancel through UTAS, or when the registration period ends.
+
+To prevent the registration of course participants from being canceled for these reasons, you need to switch the registration method from “Self-registration”, “Course registration”, or “Favorite registration” to “Register instructor”, using the function “Switch”.
```

Comment:
```suggestion
To prevent the registration of course participants from being canceled for these reasons, you need to switch the registration method from “Self-registration”, “Course registration”, or “Favorite registration” to “Register instructor”, using the function “Switch.”
```